### PR TITLE
Update default schedule time as random

### LINKED ISSUE #10094: Update dependabot default time documentation
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Configuration options for dependency updates](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates)

### What part(s) of the article would you like to see updated?

[`schedule.day`](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#scheduleday), [`schedule.time`](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#scheduletime) and [`schedule.timezone`](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#scheduletimezone) sections mention the default time for Dependabot updates being at 05:00 UTC but from the GitHub changelog post, [Dependabot now schedules version updates uniformly](https://github.blog/changelog/2021-06-16-dependabot-now-schedules-version-updates-uniformly/), each repository is now assigned a time at random.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by wongjn**
As per https://github.blog/changelog/2021-06-16-dependabot-now-schedules-version-updates-uniformly/

### Why:

Closes #10094

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@wongjn 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates.md [hunk 1]
```diff
@@ -539,7 +539,7 @@ updates:
 
 ### `schedule.day`
 
-When you set a `weekly` update schedule, by default, {% data variables.product.prodname_dependabot %} checks for new versions on Monday at 05:00 UTC. Use `schedule.day` to specify an alternative day to check for updates.
+When you set a `weekly` update schedule, by default, {% data variables.product.prodname_dependabot %} checks for new versions on Monday at a random time, set per repository. Use `schedule.day` to specify an alternative day to check for updates.
```
Comment (lecoursen) [inline content/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates.md:542]:
```suggestion
When you set a `weekly` update schedule, by default, {% data variables.product.prodname_dependabot %} checks for new versions on Monday at a random time set for the repository. Use `schedule.day` to specify an alternative day to check for updates.
```

File: content/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates.md [hunk 2]
```diff
@@ -582,7 +582,7 @@ updates:
 
 ### `schedule.timezone`
 
-By default, {% data variables.product.prodname_dependabot %} checks for new versions at 05:00 UTC. Use `schedule.timezone` to specify an alternative time zone. The time zone identifier must be from the Time Zone database maintained by [iana](https://www.iana.org/time-zones). For more information, see [List of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+By default, {% data variables.product.prodname_dependabot %} checks for new versions at a random set time for the repository. Use `schedule.timezone` to specify an alternative time zone. The time zone identifier must be from the Time Zone database maintained by [iana](https://www.iana.org/time-zones). For more information, see [List of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
```
Comment (lecoursen) [inline content/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates.md:585]:
```suggestion
By default, {% data variables.product.prodname_dependabot %} checks for new versions at a random time set for the repository. Use `schedule.timezone` to specify an alternative time zone. The time zone identifier must be from the Time Zone database maintained by [iana](https://www.iana.org/time-zones). For more information, see [List of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
```

**Review by lecoursen — COMMENTED** [review]
Thanks @wongjn, this is really helpful! I'm applying one small change to use consistent language in all three places and then I'll merge! ⚡

**Review by lecoursen — APPROVED** [review]

**Pull request merged**